templates/miq-template.yaml

```diff
@@ -88,7 +88,7 @@ objects:
             persistentVolumeClaim:
               claimName: ${NAME}
         containers:
-        - image: fbladilo/miq-app:latest
+        - image: fbladilo/miq-app:${APPLICATION_IMG_TAG}
```

Comment:
@simaishi I'm not sure, at a glance it would seem like it's redundant with ImageStream 'name' but I need to test in order to ensure.